src/core/ChannelHandler.js

```diff
@@ -0,0 +1,103 @@
+import Observable from 'zen-observable';
+
+const events = [
+  'phx_reply',
+  'phx_error',
+  'status_changed',
+  'stopped',
+  'error',
+  'failed',
+  'starting',
+  'started',
+  'deploying',
+  'deployed',
+  'deployment_failed',
+  'ready',
+  'terminated',
+  'snapshot_taken',
+  'snapshot_reverted'
```

Comment:
given we have `src/core/ChainEvent.js` isn't this just a duplicate? can't we just do a `Object.values(Event)` instead of having to maintain both?

Reply:
Will change